.goreleaser.yml

```diff
@@ -55,6 +55,6 @@ release:
     - glob: 'terraform-registry-manifest.json'
       name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
   # If you want to manually examine the release before its live, uncomment this line:
-  draft: false
```

Comment:
I think we need draft: false rather than commenting it out

Reply:
Done